### ISSUE TITLE
Add support for the --env option added in the admin CLI

### ISSUE THREAD

**Issue by lemonmade** [Shopify/argo-checkout#33]
An option was added to select a file that will be used to define environment variables that are accessible to the Argo script (both during development, and at build time). [Here's the relevant PR](https://github.com/Shopify/argo-admin-cli/pull/10).

@robin-drexler does anyone on the cross-sell team have time to pick this up? If it hasn't been requested by partners yet, it might not be a huge deal, but eventually someone will definitely want something like this.

**Comment by robin-drexler** [comment]
Yeah, we already had the discussion on our radar and were monitoring closely. Thanks for opening an issue here! 

We should be able to take care of that should it become necessary. 
Should that change, we'll certainly reach out.
